eng/versioning/version_client.txt

```diff
@@ -81,7 +81,7 @@ com.azure:azure-core-tracing-opentelemetry;1.0.0-beta.19;1.0.0-beta.20
 com.azure:azure-cosmos;4.25.0;4.26.0-beta.1
 com.azure:azure-cosmos-benchmark;4.0.1-beta.1;4.0.1-beta.1
 com.azure:azure-cosmos-dotnet-benchmark;4.0.1-beta.1;4.0.1-beta.1
-com.azure.cosmos.spark:azure-cosmos-spark_3-1_2-12;4.6.0-beta.1;4.6.0-beta.1
+com.azure.cosmos.spark:azure-cosmos-spark_3_2-12;4.6.0-beta.1;4.6.0-beta.1
```

Comment:
Don't we need 2 entries here? Are we planning to always keep these on the same version, if yes, that would be a big assumption.

Reply:
Yes - as I discussed in the PR description I want to force us to always keep the same version - otherwise it gets too confusing for customers IMO. In .Net terms I would think about this as a multi-target package - like one nuget targeting .Net 4 and .Net Core 3 etc. - here just two different Spark runtime versions - but it should be seen as one product - The Cosmos Spark Connector for Spark 3.
Keeping versions identical is essential to be able to do this. So, this way of versioning is absolutely intentional.

Reply:
Perfect!